it/assets/PaymentAllocationChargesTestConstants.scala

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package assets
+
+import models.paymentAllocationCharges.{DocumentDetail, FinancialDetail, PaymentAllocationChargesModel, SubItem}
+import play.api.libs.json.{JsValue, Json}
+
+object PaymentAllocationChargesTestConstants {
+
+
+ val documentDetail: DocumentDetail = DocumentDetail(
+  taxYear = "2018",
+  transactionId = "id",
+  documentDescription = Some("documentDescription"),
+  originalAmount = Some(300.00),
+  outstandingAmount = Some(200.00),
+  documentDate = "2018-03-29"
+ )
+
+ val documentDetail2: DocumentDetail = DocumentDetail(
+  taxYear = "2019",
+  transactionId = "id2",
+  documentDescription = Some("documentDescription2"),
+  originalAmount = Some(100.00),
```

Comment:
Payment documents always have a negative value (e.g. -100.00) This is true for total amount and outstanding amount.